extensions/2.0/Khronos/KHR_materials_sheen/schema/glTF.KHR_materials_sheen.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_sheen glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the sheen material model.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "sheenFactor": {
+            "type": "number",
+            "description": "The sheen layer intensity.",
+            "default": 0.0,
+            "minimum": 0.0,
+            "maximum": 1.0,
+            "gltf_detailedDescription": "The sheen layer intensity of the material. A value of 0.0 means the material has no sheen layer enabled."
+        },
+        "sheenColor": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0.0,
+                "maximum": 1.0
+            },
+            "minItems": 3,
+            "maxItems": 3,
+            "description": "Color of the sheen layer.",
+            "default": [1, 1, 1]
+        },
+        "sheenTexture": {
+            "allOf": [ { "$ref": "textureInfo.schema.json" } ],
+            "description": "The sheen layer texture.",
+            "gltf_detailedDescription": "The sheen layer texture. Stored in channel RGB the sheen color in Gamma and in A the linear factor."
```

Comment:
Do we have any existing references on efficiently packing sheen color and sheen factor?

On one hand, a single texture is elegant. On the other hand, I'm not sure whether one PNG is always going to be more efficient than two JPGs. Or whether this packing scheme will remain useful with `KHR_texture_basisu`.

Reply:
Yup I totally agree, and the main intent for us is to reduce the overall number of samplers due to the small allowed number on some devices.

Reply:
For consistency, please replace "Gamma" with "sRGB transfer function".

Reply:
thanks @lexaknyazev I have just updated it